File: coffee-shop-backend/coffee-shop-backend/Entities/Creditcard.cs
using System;
using System.Collections.Generic;

namespace coffee_shop_backend.Entities;

public partial class Creditcard
{
    public string CreditCardNumber { get; set; } = null!;

    public string NameAndSurname { get; set; } = null!;

    public string Cvc2 { get; set; } = null!;

    public string ValidUntil { get; set; } = null!;

    public decimal AccountState { get; set; }
}

File: coffee-shop-ui/src/app/models/credit-card.ts
export class CreditCard {
    NameAndSurname: string;
    CreditCardNumber: string;
    Cvc2: string;
    ValidUntil: string;
    AccountState: number;
    constructor(
        NameAndSurname: string,
        CreditCardNumber: string,
        Cvc2: string,
        ValidUntil: string,
        AccountState: number,
    ) {
        this.NameAndSurname = NameAndSurname;
        this.CreditCardNumber = CreditCardNumber;
        this.Cvc2 = Cvc2;
        this.ValidUntil = ValidUntil;
        this.AccountState = AccountState;
    }
}
